python/ray/tune/trainable/trainable.py

```diff
@@ -1395,7 +1396,7 @@ def save_checkpoint(self, checkpoint_dir: str) -> Optional[Union[str, Dict]]:
 
         Returns:
             A dict or string. If string, the return value is expected to be
-            prefixed by `checkpoint_dir`. If dict, the return value will
+            the `checkpoint_dir`. If dict, the return value will
```

Comment:
Maybe we should just make the return type `Optional[Dict]` since we already have the string

Reply:
Good idea, will update